Assets/Scripts/Actors/ActorComponents/ActorPhysics.cs

```diff
@@ -43,113 +48,135 @@ public float groundedMoveSpeed
 	[SerializeField] protected float jumpMoveSpeed = 6f;
 	[SerializeField] protected float rollMoveSpeed = 6f;
 	[SerializeField] protected float climbMoveSpeed = 6f;
-
-	// Model Info
-	[Space(10)][Header("Model Info")]
-	public Transform model;
+	#endregion
+
+	#region Collisions
+	[Space( 10 ), Header( "Collisions" )]
+	public Collider bumper;
+	public Collider lifter;
+	public Collider climbBumper;
+	private Transform _bumperTransform;
+	private Rigidbody _bumperRigidbody;
+	#endregion
+
+	#region Model Info
+	[Space( 10 ), Header( "Model Info" )]
 	[SerializeField] float modelTurnSpeed = 7f;
-	protected Vector3 modelOffset;
-
-	// Slope Checking
-	[Space(10)][Header("Slope Checking")]
-	[SerializeField] float groundSlopeCheckRadius = 0.2f;
-	[SerializeField] float groundSlopeCheckHeight = 0.7f;
-	float groundSlopeCapsuleWidth = 0.4f;
-	float groundSlopeCastDist = 0.3f;
+	[Range( 0.0f, 1.0f )]
+	[SerializeField] float lookDirLerp = 0.002f;
 
-	float groundSlopeFallSpeed = -30f;
+	private Vector3 _lastPos;
+	#endregion
 
-	protected float groundSlopeSpeedMod = 1f;
-	float groundSlopeSpeedModeDelta = 2.5f;
-
-	[SerializeField] MinMaxF slopeLimits = new MinMaxF( 0.28f, 0.72f );
-
-	// Jumping
-	[Space(10)][Header("Jumping")]
+	#region Jumping
+	[Space( 10 ), Header( "Jumping" )]
 	[SerializeField] public float jumpForce = 8.5f;
 
-	[SerializeField] float jumpCheckDistance = 1.3f;
-	[SerializeField] Vector3 groundPosOffset = new Vector3(0f, -1f, 0f);
-	
-	[SerializeField] float jumpCheckRadius = 1.2f;
+	[SerializeField] float jumpCheckDistance = 1.0f;
+	[SerializeField] float jumpCheckRadius = 0.5f;
+
 	[SerializeField] LayerMask jumpLayer = 0;
 	[SerializeField] float minJumpDot = 0.4f;
```

Comment:
This could probably have a better name. It's clear if dot product is second nature, but totally obtuse otherwise.